kubectl-kuttle

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
it also makes sense to convert the script to a plain posix shell. e.g. `#!/bin/sh` and add an `-e` flag to stop in case of errors: https://www.grymoire.com/Unix/Sh.html#uh-55